openstudiocore/src/energyplus/CMakeLists.txt

```diff
@@ -292,6 +292,7 @@ set(${target_name}_src
   ForwardTranslator/ForwardTranslateSteamEquipment.cpp
   ForwardTranslator/ForwardTranslateSubSurface.cpp
   ForwardTranslator/ForwardTranslateSurface.cpp
+  ForwardTranslator/ForwardTranslateSurfacePropertyConvectionCoefficients.cpp
```

Comment:
This object would be a candidate for reverse translation, not saying you have to do it only that it would be possible